packages/nextjs/package.json

```diff
@@ -34,7 +34,7 @@
   },
   "peerDependencies": {
     "next": "^10.0.8 || ^11.0 || ^12.0",
-    "react": "15.x || 16.x || 17.x",
+    "react": "15.x || 16.x || 17.x || 18.x",
```

Comment:
I know we're trying to get this out the door, but I think this should go in a separate PR.

Reply:
Sure

Reply:
https://github.com/getsentry/sentry-javascript/pull/4831